SteamPrefill/SteamManager.cs

```diff
@@ -346,5 +348,78 @@ private async Task<BenchmarkWorkload> BuildBenchmarkWorkloadAsync(List<uint> app
         }
 
         #endregion
+
+        #region Status
+
+        public async Task CurrentlyDownloadedAsync(SortOrder sortOrder, string sortColumn)
+        {
+            await _cdnPool.PopulateAvailableServersAsync();
+
+            List<uint> appIds = LoadPreviouslySelectedApps();
+            ByteSize totalSize = new ByteSize();
+            Dictionary<string, ByteSize> index = new Dictionary<string, ByteSize>();
+
+            var timer = Stopwatch.StartNew();
+            _ansiConsole.LogMarkupLine("Loading Manifests");
+            foreach (uint appId in appIds)
+            {
+                AppInfo appInfo = await _appInfoHandler.GetAppInfoAsync(appId);
+
+                var filteredDepots = await _depotHandler.FilterDepotsToDownloadAsync(_downloadArgs, appInfo.Depots);
+                await _depotHandler.BuildLinkedDepotInfoAsync(filteredDepots);
+
+                var manifests = await _manifestHandler.GetAllManifestsAsync(filteredDepots);
+
+                var size = ByteSize.FromBytes(manifests.Sum(s1 => s1.Files.Sum(s2 => s2.Chunks.Sum(s3 => s3.CompressedLength))));
```

Comment:
You can make things a bit simpler here, there are some methods that already exist that can take care of this for you.  Also there is a bit more logic on how Steam determines the download size that is being missed here.  

Try this out:
```
var allChunksForApp = await _depotHandler.BuildChunkDownloadQueueAsync(filteredDepots);
var size = ByteSize.FromBytes(allChunksForApp.Sum(e => e.CompressedLength));
```

This also means that the manifests variable above is unnecessary, since `BuildChunkDownloadQueueAsync` takes care of that.